<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>879</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>971</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="departments">
     <attribute name="title">
      <string>departments</string>
     </attribute>
     <widget class="QPushButton" name="get_deps_button">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>get departments</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableDep">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="NewDepBut">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new department</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NewDepEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>90</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Personal">
     <attribute name="title">
      <string>Personal</string>
     </attribute>
     <widget class="QTableWidget" name="tablePer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>751</width>
        <height>192</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="get_per_button">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>80</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Get personal</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Faces">
     <attribute name="title">
      <string>Faces</string>
     </attribute>
     <widget class="QTableWidget" name="tableFaces">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="get_faces_button">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Faces</string>
      </property>
     </widget>
     <widget class="QPushButton" name="get_face_image_but">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Get face image</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_face_id">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>30</y>
        <width>48</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="face_image">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>90</y>
        <width>391</width>
        <height>331</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Cameras">
     <attribute name="title">
      <string>Cameras</string>
     </attribute>
     <widget class="QPushButton" name="get_cameras_button">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Get cameras</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableCam">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>871</width>
        <height>192</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="Cabinets">
     <attribute name="title">
      <string>Cabinets</string>
     </attribute>
     <widget class="QTableWidget" name="tableCabinets">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>491</width>
        <height>192</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="get_cabinets_button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Get cabinets</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="In_out">
     <attribute name="title">
      <string>In_out</string>
     </attribute>
     <widget class="QTableWidget" name="tableInOut">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>381</width>
        <height>192</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="get_inout_button">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Get In-Out</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalInOutLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="file_path_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>800</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>750</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>choose buttnon</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
